bin/reth/src/lib.rs

```diff
@@ -16,8 +16,8 @@
 //!   and leak detection functionality. See [jemalloc's opt.prof](https://jemalloc.net/jemalloc.3.html#opt.prof)
 //!   documentation for usage details. This is **not recommended on Windows**. See [here](https://rust-lang.github.io/rfcs/1974-global-allocators.html#jemalloc)
 //!   for more info.
-//! - `only-info-logs`: Disables all logs below `info` level. This can speed up the node, since
-//!   fewer calls to the logging component is made.
+//! - `min-info-logs`: Disables all logs below `info` level. This can speed up the node, since fewer
```

Comment:
please add the remaining features here just like this one so it is documented

Reply:
@onbjerg Done